source/setup/nao_setup.rst

```diff
@@ -21,117 +45,179 @@ Download opn file
 
 .. tip::
 
-    ``<hostname>`` is likely ``nao.local`` for new or factory reset robots.
+    It's sometimes useful to just flash the base opn image if you experience issues with robot kinematics, but are confident there are no hardware issues.
+    You can simply use the base image with the aforementioned flash methods to restore the default nao image, and then flash the custom image once you confirm
+    things roughly work as expected on the base image.
+
+.. tip::
 
-    Robots that are flashed without factory reset should retain their hostnames and you don't
-    need to specify ``-h <hostname>`` and ``<robot-name>.local`` will be used automatically.
+    ``<hostname>`` is likely ``nao.local`` for new or factory reset robots. This could be used instead of the IP address during setup.
 
 .. tip::
 
-    Workaround: If this last step is causing trouble for you, try changing the hostname at 
+    Workaround: If this last step is causing trouble for you, try changing the hostname at
     the command line first, then syncing, for example:
-    
+
 .. code-block:: bash
-    
+
     PC$ ssh nao@nao.local
     nao$ sudo nano /etc/hostname
     # Agree, then change the hostname from 'nao' to for example 'treebeard'
     # then reboot the robot and continue as normal, i.e.
     PC$ nao_sync -s treebeard
     PC$ nao_sync -rd treebeard
     # reboot again
-    
+
 *********************************
 Connecting to GameController Wifi
 *********************************
 
-To play a game by the SPL Rules, the `runswift` executable needs to send packets of 
-information to, and respond to commands from, one specific soccer field's 
+To play a game by the SPL Rules, the ``runswift`` executable needs to send packets of
+information to, and respond to commands from, one specific soccer field's
 `GameController <https://github.com/RoboCup-SPL/GameController3/>`_. Typically the
-competition organisers will provide the list of field SSIDs and any other details 
-at the competition, for example `SPL_A` to `SPL_E` has been typical of RoboCup.
+competition organisers will provide the list of field SSIDs and any other details
+at the competition, for example ``SPL_A`` to ``SPL_E`` has been typical of RoboCup.
 
-One way to set this up is to SSH into the robot and use the change field script, this is 
-analagous to connecting to a wifi hotspot, one needs to specify things like the 
-`SSID <https://www.lifewire.com/definition-of-service-set-identifier-816547>`_:
+One way to do this is to use the change_field script located in bin.
+You can also modify the WIFI network manually in the ``/etc/netplan`` directory and run ``sudo netplan apply`` if you're in a pinch.
 
 .. code-block:: bash
 
-    PC$ ssh nao@treebeard.local
-    # sudo bin/changeField.py <SSID>
-    sudo bin/changeField.py SPL_E
+    bin/change_field.py <robot hostname or ip> <field (e.g. SPL_A)>
+
+You can also provide ``*`` to change_field to change the field of all robots in robots.cfg.
```

Comment:
```suggestion
You can also provide ``all`` to change_field to change the field of all robots in robots.cfg.
```

Reply:
I actually like change-field more as I think about it, because then it's not specific to wifi if the SPL changes to ultrasound-based comms, for example.

Reply:
I'm going to make it change-wifi since that's unlikely to happen for awhile, and change-wifi is more straight forward. We can explore changing it back if this does happen